.github/workflows/ubuntu.yaml

```diff
@@ -67,8 +67,12 @@ jobs:
         sleep 15
       working-directory: test
 
+    # Run as sudo to be able to bind to DHCP client port, then reset
+    # permissions to code coverage files if necessary
     - name: Run tests
-      run: sudo ctest --verbose -C RelWithDebInfo
+      run: |
+        sudo ctest --verbose -C RelWithDebInfo
+        sudo chmod -R 777 .
```

Comment:
Should be `sudo chmod -R a+rw .` No need to make all files executable. Directories are executable by default.

Reply:
Fixed.